airflow/providers/google/cloud/example_dags/example_bigtable.py

```diff
@@ -105,7 +105,7 @@
         instance_type=int(CBT_INSTANCE_TYPE),
         instance_labels=json.loads(CBT_INSTANCE_LABELS),
         cluster_nodes=int(CBT_CLUSTER_NODES),
-        cluster_storage_type=int(CBT_CLUSTER_STORAGE_TYPE),
+        cluster_storage_type=int(int(CBT_CLUSTER_STORAGE_TYPE)),
```

Comment:
Is it needed?

Reply:
Of course it is mistake. Thank you for pointing this out